emcc.py

```diff
@@ -2726,6 +2726,9 @@ def do_binaryen(target, options, wasm_target):
     webassembly.add_emscripten_metadata(wasm_target)
 
   if final_js:
+    if shared.Settings.LE_HEAP or sys.byteorder == 'big':
```

Comment:
I don't think we should look at `sys.byteorder`. The build should be the same no matter on what machine it runs, so that a user can build for either target no matter where they are.

Reply:
Removed sys.byteorder